package.json

```diff
@@ -32,8 +32,8 @@
     "npm": ">=9"
   },
   "volta": {
-    "node": "18.16.0",
-    "npm": "9.6.7"
+    "node": "18.17.1",
+    "npm": "10.0.0"
   },
```

Comment:
Is this necessary for this change?

Reply:
See https://github.com/thefrontside/simulacrum/pull/267#discussion_r1318876740